src/shared/extensionGlobals.ts

```diff
@@ -115,6 +115,7 @@ interface ToolkitGlobals {
     codelensRootRegistry: CodelensRootRegistry
     resourceManager: AwsResourceManager
     uriHandler: UriHandler
+    hostname: string
```

Comment:
Something like this:

```suggestion
    /** An id to differentiate the current machine being run on. Can help distinguish a remote from a local machine.  */
    machineId: string
```